src/test/java/core/basesyntax/strategy/PurchaseHandlerTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.strategy.handler.impl.PurchaseHandler;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class PurchaseHandlerTest {
+    private PurchaseHandler purchaseHandler;
+
+    @BeforeEach
+    public void setUp() {
+        purchaseHandler = new PurchaseHandler();
+    }
+
+    @AfterEach
```

Comment:
Try to be consistent: in some test classes you clear the Storage by using AfterEach annotation, in some classes by using BeforeEach annotation